wagtail_streamfield_migration_toolkit/test/tests/test_block_structure.py

```diff
@@ -15,19 +17,20 @@
 class FieldChildBlockTest(TestCase):
     """Changes to `char1`"""
 
-    @classmethod
-    def setUpTestData(cls):
+    def setUp(self):
```

Comment:
Why have you changed away from `setUpTestData` here?

Reply:
At some point I had a problem with data getting changed between testcases. I think it should work fine now though, I could add a commit for it if it does